index.md

```diff
@@ -1,5 +1,39 @@
 ---
-title: About JuMP
+layout: page
 ---
 
-hello, world
+![jump](/assets/jump-logo-with-text-large.svg)
+
+JuMP is a modeling language and supporting packages for mathematical
+optimization in [Julia](https://julialang.org).
+
+## Getting started
+
+To get started, read the [Installation Guide](https://www.juliaopt.org/JuMP.jl/stable/installation/)
+in the JuMP documentation.
+
+Once you've installed JuMP, a good next step is the [Quickstart Guide](https://www.juliaopt.org/JuMP.jl/stable/quickstart/).
+
+There are also a number of tutorials available at [JuMPTutorials.jl](https://github.com/JuliaOpt/JuMPTutorials.jl).
+
+If you need help, search the [Forum](https://discourse.julialang.org/c/domain/opt)
+for similar questions, or open a new one.
+
+---
+
+##### 4th annual JuMP-dev
```

Comment:
We should use heading depths correctly. `#####` is a sub-heading under `####`. If the issue is the font size of `##`, then that can be addressed with a change to the css or html.